packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -135,49 +141,53 @@ const Header: React.FC<Props> = ({
 
   const renderUserActions = () => {
     if (!canLogin || breakpoint <= Breakpoint.sm) return null;
-
+    // FEAT:: back to main account cta if oauth mode
```

Comment:
Did you consider creating custom component `CXLHeader` based on the `Header` instead of heavily modifying the upstream one?

Reply:
yeah, sounds good, should have thought like others